packages/jupyterlab-ws-chat/pyproject.toml

```diff
@@ -30,7 +30,7 @@ dependencies = [
     "jupyterlab~=4.0",
     "traitlets>=5.0",
     "deepmerge>=1.0",
-    "langchain"
+    "pydantic>=2"
```

Comment:
Requiring `>=2` as per: https://docs.pydantic.dev/latest/migration/#continue-using-pydantic-v1-features

> The Pydantic V2 package also continues to provide access to the Pydantic V1 API by importing through pydantic.v1. For example, you can use the BaseModel class from Pydantic V1 instead of the Pydantic V2 pydantic.BaseModel class: